airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/DebeziumOperations.kt

```diff
@@ -42,3 +42,10 @@ interface CdcPartitionReaderDebeziumOperations<T : Comparable<T>> {
     /** Tries to extract the WAL position from a [SourceRecord]. */
     fun position(sourceRecord: SourceRecord): T?
 }
+
+/** [DeserializedRecord]s are used to generate Airbyte RECORD messages. */
+data class DeserializedRecord(
+    val streamID: StreamIdentifier,
+    val data: ObjectNode,
+    val changes: Map<Field, FieldValueChange>,
```

Comment:
it seems `changes` are always empty. If a record is UPDATE it will be reflected in the data but not in this field. So what is this field for?

Reply:
`changes` are indeed not set outside of that one unit test, but I can see a case where `deserialize` might do some simple data transformations (like decode some base64, parse some timestamps, etc) which might fail. Do you think we should remove this field?

Reply:
Got it. I'm okay either way (leaving changes here, or having those transformations defined somewhere else like we have for debezium operations like this https://github.com/airbytehq/airbyte/pull/46929/files#diff-d8ac0f6a4b48e907dedddc9dbd560381a49af3b99b4064a48c32deda3594a30d)